nest-forms-backend/src/forms/subservices/forms-task.subservice.ts

```diff
@@ -56,9 +56,16 @@ export default class FormsTaskSubservice {
     // Delete the forms along with its files
     await this.filesService.deleteFileMany(fileIds)
     const deleted = await this.prismaService.forms.deleteMany({
+      // Delete the forms based on the same where clause as above, not by IN: formIds.
+      // The reason is that there is a LIMIT on how many ids can be passed to IN, this ensures that all forms are deleted.
+      // The result of WHERE is the same as in the findMany query above, since oneWeekAgo is a constant.
       where: {
```

Comment:
I would extract where to the begging of the function. Not because of DRY, but because it critical functions like this it is worth to have extra guarantee the delete is the same as the find.